x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/telemetry/task_based/all_types.ts

```diff
@@ -43,7 +44,7 @@ export default ({ getService }: FtrProviderContext) => {
       await deleteAllExceptions(supertest, log);
     });
 
-    it('should only have task metric values when no rules are running', async () => {
+    it('@skipInQA should only have task metric values when no rules are running', async () => {
```

Comment:
would be nice to have comments about the reason we disable tests in certain environments

Reply:
Upon encountering failing tests during the MKI/QA environment tests, we incorporate the skipInQA tag. Subsequently, we associate the PR that contains the skipped tests in QA with this particular [ticket](https://github.com/elastic/kibana/issues/169770) so that we can fix them later.